### ISSUE TITLE
Attribute multiplicity is ignored for relationship class relations

### ISSUE THREAD

**Issue by kshychko** [gs-gs/staruml-cefact#5]
Multiplicity set to 0..* doesn't produce a property of array type as expected, but a property of object type. It might happens, because the multiplicity is set not in the `UMLAssociationClassLink`, but in the corresponding `UMLAssociation` object

**Comment by faizanvahevaria** [comment]
@kshychko Please review this issue. If all good, then let's close this issue

**Comment by kshychko** [comment]
@faizanvahevaria, isn't fixed to me, getting the same result.

**Comment by faizanvahevaria** [comment]
@kshychko 
You talk about 

**Aggregation**:

TransportMeans:
          allOf:
            - $ref: '#/components/schemas/TransportMeansIds'
            - type: object

**RelationshipClass**:

TransportMovementParty:
          allOf:
            - $ref: '#/components/schemas/TransportPartyIds'
            - $ref: '#/components/schemas/TransportMovementParty'
            - type: object

Right?

**Comment by kshychko** [comment]
Yes, when I set multiplicity to 1..* for I expect to get the following
```
	TransportMovementParty:
          type: array
          minItems: 1
          items:
            allOf:
              - $ref: '#/components/schemas/TransportPartyIds'
              - $ref: '#/components/schemas/TransportMovementParty'
              - type: object
```

At the moment I'm getting:

```
        TransportMovementParty:
          allOf:
            - $ref: '#/components/schemas/TransportPartyIds'
            - $ref: '#/components/schemas/TransportMovementParty'
            - type: object
```

**Comment by kshychko** [comment]
works fine, except for '1..*' minItems=1 is not added

**Comment by kshychko** [comment]
Fixed with 51d7ecf212ec221a4c11723e6eeeb80b068220d4
